lookout/style/format/virtual_node.py

```diff
@@ -117,6 +117,14 @@ def from_node(node: bblfsh.Node, file: str, path: str,
         """
         outer_token = file[node.start_position.offset:node.end_position.offset]
         if not node.token:
+            if outer_token == "''" or outer_token == '""':
```

Comment:
We are going to have fun time porting to other languages which distinguish between single and double quotes...

Reply:
We'll also have a lot of fun when we'll port to languages with `"""` and other quoting sequences because bblfsh doesn't help us at all in this case to know where the delimiters are so we'll have to go full-heuristic.